pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -169,14 +182,54 @@ func (s *Service) RegisterInstanceWithAPIServerELB(i *infrav1.Instance) error {
 	return nil
 }
 
-// GenerateELBName generates a formatted ELB name
-func GenerateELBName(clusterName string, elbName string) string {
-	return fmt.Sprintf("%s-%s", clusterName, elbName)
+func GenerateELBName(clusterName string, elbName string) (string, error) {
```

Comment:
Please restore the godoc and adjust it based on the changes you're making in this PR.